src/pages/Landing/components/StudentTicket/History/History.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-nested-ternary */
+import React, { useState } from 'react'
+import attachment from '../../../../../assets/images/attachment.png'
+import styles from './History.module.scss'
+import head from './mockData'
+import deleteIcon from '../../../../../components/icons/icons-svg/delete.svg'
+
+const History = () => {
+  return (
+    <div className={styles.History}>
+      <h4 className={styles.HistoryText}>Report History</h4>
```

Comment:
![image](https://user-images.githubusercontent.com/43199095/112511871-68b99a80-8dc5-11eb-891d-dfecf7486c4d.png)
Header should be fixed when scrolling down